services/dkg/pedersen/mod_test.go

```diff
@@ -1,281 +1,460 @@
 package pedersen
 
 import (
+	"encoding/hex"
+	"encoding/json"
 	"testing"
 
 	"github.com/dedis/d-voting/internal/testing/fake"
-	"github.com/dedis/d-voting/services/dkg"
-	"github.com/dedis/d-voting/services/dkg/pedersen/types"
+	// "github.com/dedis/d-voting/services/dkg"
+	// "github.com/dedis/d-voting/services/dkg/pedersen/types"
 	"github.com/stretchr/testify/require"
-	"go.dedis.ch/dela/crypto"
-	"go.dedis.ch/dela/crypto/ed25519"
-	"go.dedis.ch/dela/mino"
-	"go.dedis.ch/dela/mino/minogrpc"
-	"go.dedis.ch/dela/mino/router/tree"
-	"go.dedis.ch/kyber/v3"
+	// "go.dedis.ch/dela/crypto"
+	// "go.dedis.ch/dela/crypto/ed25519"
+	"go.dedis.ch/dela/core/store/kv"
+	// "go.dedis.ch/dela/mino"
+	// "go.dedis.ch/dela/mino/minogrpc"
+	// "go.dedis.ch/dela/mino/router/tree"
+	// "go.dedis.ch/kyber/v3"
 )
 
-func TestPedersen_Listen(t *testing.T) {
-	pedersen, _ := NewPedersen(fake.Mino{}, false)
+// // After initializing a Pedersen when dkgMap is empty, the actors map should be empty
+// func TestPedersen_InitEmptyMap(t *testing.T) {
 
-	actor, err := pedersen.Listen()
-	require.NoError(t, err)
+// }
 
-	require.NotNil(t, actor)
-}
+// // After initializing a Pedersen when dkgMap is not empty, the actors map should contain the same information
+// // as dkgMap
+func TestPedersen_InitNonEmptyMap(t *testing.T) {
```

Comment:
I'm missing a test that does multiple inits and shows that multiple actors were created, one per election. That's the whole point :)